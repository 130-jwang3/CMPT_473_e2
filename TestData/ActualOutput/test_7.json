[{"id\tname\tage\tcity":"2\t\"Jane, Smith\"\t30\t\"San Francisco\""}]