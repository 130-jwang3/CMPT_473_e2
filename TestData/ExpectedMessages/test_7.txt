Converting TestData/TestFiles/test_7.csv -> TestData/ActualOutput/test_7.json [sep=     , header=0]
Only keeping rows: 1 2
   id         name  age           city
1   2  Jane, Smith   30  San Francisco
