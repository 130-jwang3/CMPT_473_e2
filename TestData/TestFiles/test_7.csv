id	name	age	city
1	"John"	25	"New York"
2	"Jane, Smith"	30	"San Francisco"
3	"Tom"	22	"Los Angeles"