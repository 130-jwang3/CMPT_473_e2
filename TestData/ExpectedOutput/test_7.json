[{"id": "2", "name": "Jane, Smith", "age": 30, "city": "San Francisco"}]
